From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ppatierno__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

rmca14 commented:
@jeffstokes72 
Please review this PR and note your approval/disapproval here. Thanks.

jeffstokes72 commented:
We are looking into this thank you!

From: Ross McAllister [mailto:notifications@github.com]
Sent: Tuesday, August 4, 2015 12:21 PM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Jeff Stokes <Jeff.Stokes@microsoft.com>
Subject: Re: [azure-content] Limit of 5 readers per partition (#4159)


@jeffstokes72<https://github.com/jeffstokes72>
Please review this PR and note your approval/disapproval here. Thanks.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/4159#issuecomment-127663706>.

jeffstokes72 commented:
This change is approved.

From: Ross McAllister [mailto:notifications@github.com]
Sent: Tuesday, August 4, 2015 12:21 PM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Jeff Stokes <Jeff.Stokes@microsoft.com>
Subject: Re: [azure-content] Limit of 5 readers per partition (#4159)


@jeffstokes72<https://github.com/jeffstokes72>
Please review this PR and note your approval/disapproval here. Thanks.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/4159#issuecomment-127663706>.

SychevIgor committed:
Classic portal no longer supported by ML Workspace

Classic portal no longer supported by ML Workspace

MarkMcGeeAtAquent committed:
Merge pull request #12 from MicrosoftDocs/master

merge

MarkMcGeeAtAquent committed:
SEO rule violation fixes

MarkMcGeeAtAquent committed:
updates

MarkMcGeeAtAquent committed:
Merge branch 'master' into table-api-quickstarts

MarkMcGeeAtAquent committed:
quickstart updates

MarkMcGeeAtAquent committed:
Merge branch 'table-api-quickstarts' of https://github.com/MarkMcGeeAtAquent/azure-docs-pr into table-api-quickstarts

MarkMcGeeAtAquent committed:
updates

MarkMcGeeAtAquent committed:
Update create-table-dotnet.md

MarkMcGeeAtAquent committed:
art fix

MarkMcGeeAtAquent committed:
Merge branch 'table-api-quickstarts' of https://github.com/MarkMcGeeAtAquent/azure-docs-pr into table-api-quickstarts

MarkMcGeeAtAquent committed:
add red callout to Create button

ShawnJackson committed:
edit pass: stream-analytics-out-of-order-and-late-events

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into stream-analytics-out-of-order-and-late-events

ShawnJackson committed:
edit pass: stream-analytics-out-of-order-and-late-events

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into stream-analytics-out-of-order-and-late-events

ShawnJackson committed:
edit pass: stream-analytics-out-of-order-and-late-events

ShawnJackson committed:
edit pass: stream-analytics-out-of-order-and-late-events

gled4er committed:
Fixing usings for C# samples.

kadimitr committed:
Improving usings and fixing attribute explanations.

kadimitr committed:
More changes following the comments from the ticket.

paulathurman committed:
edit pass: scheduled-events

paulathurman committed:
edit pass: scheduled-events

kadimitr committed:
Adding clarification for Cosmos DB Trigger currernly firing only one of mutiple functions listening to the same collection.

ShawnJackson committed:
Merge branch 'master' into stream-analytics-out-of-order-and-late-events

ShawnJackson committed:
Update stream-analytics-out-of-order-and-late-events.md

paulathurman committed:
edit pass: active-directory-saas-vodeclic-tutorial

ShawnJackson committed:
Update stream-analytics-out-of-order-and-late-events.md

ShawnJackson committed:
Update stream-analytics-out-of-order-and-late-events.md

Commit:
Add a How-To file for managing resources using ARM templates

Commit:
Fix some style issues flagged by acrolinx

Commit:
Fix two broken links.

gled4er committed:
Fixing code review issues.

Commit:
Try adding .md extension to correct validation warnings about links.

Commit:
Try adding preceding .md to topic links.

twooley committed:
updated screenshots

mimig1 committed:
Cosmos DB: Renaming documentdb files to sql-api.

mimig1 committed:
update

Commit:
automation updates

ShawnJackson committed:
Merge branch 'master' into stream-analytics-out-of-order-and-late-events

ShawnJackson committed:
Update stream-analytics-out-of-order-and-late-events.md

Commit:
Shorten TOC link

billmath committed:
updating

billmath committed:
updating

Commit:
Remove the json body of the template from the doc.

mimig1 committed:
fix validation errors

Commit:
Update link to template to point to the raw json on GitHub.

mimig1 committed:
update

mimig1 committed:
art file updates

amanbha committed:
Updating code snippet for V2 Remoting Custom serializer

itechedit committed:
edit pass: active-directory-windesktop

mimig1 committed:
art updates

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-setup.md

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-use.md

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-configure.md

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-test.md

gled4er committed:
Fix identation issues and correct spelling of function.json parameter for lease collection throughput.

v-viinde committed:
USER STORY 1144920 SaaS App Tutorial: Proofpoint on Demand update

v-viinde committed:
Merge branch 'master' into proofpointupdate

billmath committed:
updating links

dominicbetts committed:
Update CF FAQ with map info

Commit:
fix for @isaco88

dominicbetts committed:
CF configure article

daveirwin1 committed:
Updated for 9.3.0

cephalin committed:
merger feedback

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-introduction.md

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-introduction.md

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-setup.md

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-test.md

Commit:
Apply PR feedback

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-use.md

billmath committed:
updating

mawah committed:
Fix breaking change in computetarget attach

mimig1 committed:
removed unused files

tfitzmac committed:
removed duplicated content

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-configure.md

mawah committed:
Update scenario-aerial-image-classification.md

tfitzmac committed:
fixed links

mawah committed:
More realistic estimate of data copy duration

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-configure.md

Commit:
Data disk freshness

Commit:
acrolinx

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-introduction.md

iainfoulds committed:
Freshness updates and revisions

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-introduction.md

iainfoulds committed:
Updating SP command

Commit:
Adding alias content.

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-introduction.md

Commit:
Update

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasalias

itechedit committed:
Update active-directory-develop-guidedsetup-windesktop-setup.md

mawah committed:
Update cluster creation command (new AFS attachment method)

Commit:
Update

ktoliver committed:
Merge pull request #29965 from cynthn/attach-disk-1217

Data disk freshness

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasalias

PRMerger13 committed:
Merge pull request #29946 from mawah/patch-6

Fix breaking change in computetarget attach

iainfoulds committed:
Freshness updates on Linux SSL doc

Commit:
Updated ToC

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasalias

rmca14 committed:
Merge pull request #29949 from tfitzmac/1213delete

removed duplicated content

twooley committed:
updated screenshots

v-thepet committed:
broken link

iainfoulds committed:
Freshness updates to Windows CSE doc

ktoliver committed:
Merge pull request #29860 from amanbha/master

Updating code snippet for V2 Remoting Custom serializer

HeidiSteen committed:
edits

Commit:
update embed with button ex

HeidiSteen committed:
edits

tynevi committed:
Merge pull request #29674 from sandshadow/master

Add a How-To file for managing resources using ARM templates

Commit:
fix

billmath committed:
updating note

twooley committed:
updated screenshots

PRMerger20 committed:
Merge pull request #29969 from iainfoulds/vmfreshness121317-batch2

VM freshness updates

PRMerger16 committed:
Merge pull request #29974 from v-thepet/newfix

broken link

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta

HeidiSteen committed:
edits

Commit:
Merge pull request #29975 from jluk/master

update embed with button ex

nitinme committed:
Adding info about enterprise security package

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

iainfoulds committed:
Freshness checks and updates

PRMerger12 committed:
Merge pull request #29976 from HeidiSteen/heidist-ta

edits

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into scheduled-events

Commit:
Merge pull request #29981 from iainfoulds/vmfreshness121317-batch3

VM freshness updates

ktoliver committed:
Merge pull request #29832 from twooley/twdelegated

updated screenshots

ShawnJackson committed:
Update active-directory-saas-vodeclic-tutorial.md

ktoliver committed:
Merge pull request #29890 from v-viinde/proofpointupdate

USER STORY 1144920 SaaS App Tutorial: Proofpoint on Demand update

ShawnJackson committed:
Update active-directory-develop-guidedsetup-windesktop-use.md

nitinme committed:
validation fix

ShawnJackson committed:
Merge pull request #29661 from paulathurman/active-directory-saas-vodeclic-tutorial

edit pass: active-directory-saas-vodeclic-tutorial

paulathurman committed:
edit pass: scheduled-events

ShawnJackson committed:
Merge pull request #29869 from itechedit/active-directory-windesktop

edit pass: active-directory-windesktop

Commit:
Merge pull request #29980 from nitinme/nitinme-hdi-updates

Info about enterprise security package

Commit:
automation author updates 7

Commit:
automation : update author 8

Commit:
updates for acrolinx

kadimitr committed:
Adding example to few titles.

Commit:
Linux custom image tutorial freshness

ShawnJackson committed:
Merge pull request #29564 from paulathurman/scheduled-events

edit pass: scheduled-events

Commit:
Merge pull request #29986 from georgewallace/autoauthorupdates

automation author updates 7

Commit:
Update index.md

mimig1 committed:
updates

mimig1 committed:
fixed redirects

Lisaco88 committed:
Merge pull request #29473 from gled4er/kaniodimitrov/azure-functions-cosmosdb-additional-changes

Azure Functions Cosmos DB Binding  Additional Changes

Commit:
Update index.md

pvrk committed:
Update backup-azure-mars-troubleshoot.md

Removing "error" from the heading to make it more searcheable

pvrk committed:
Update backup-azure-mars-troubleshoot.md

spelluru committed:
No need of "ExpressioN"

shengcMSFT committed:
Update data-factory-use-custom-activities.md

remove OnDemand HDInsight linked service part as HDInsight is moving to Linux OS by default. We should always suggest running dotnet custom activity on Batch.

Commit:
Merge pull request #30000 from spelluru/adfqs1213

No need of "ExpressioN"

Lisaco88 committed:
Merge pull request #29995 from Kaiqb/master

Fix broken link

PRMerger13 committed:
Merge pull request #29999 from pvrk/patch-25

Update backup-azure-mars-troubleshoot.md

nitinme committed:
minor fix

PRMerger15 committed:
Merge pull request #30004 from nitinme/nitinme-hdi-updates

minor fix

Commit:
Merge pull request #30001 from shengcMSFT/patch-17

Update data-factory-use-custom-activities.md

linda33wj committed:
Remove column mapping constraint for SQL DW PolyBase

linda33wj committed:
Refine store cred in AKV description

PRMerger16 committed:
Merge pull request #30010 from linda33wj/master

Update ADF copy content

Commit:
images not used in article

Commit:
images not used in article

Commit:
images not used in article

dominicbetts committed:
Address feedback from Hans

dominicbetts committed:
Additional change

dominicbetts committed:
Address feedback from Hans

Commit:
quixfix01

PRMerger10 committed:
Merge pull request #30015 from MarkusVi/quixfix01

quickfix01

Commit:
Merge pull request #29921 from cephalin/zip

merger feedback

dominicbetts committed:
Updates to CF customize article

billmath committed:
updating

billmath committed:
updating

cephalin committed:
cli link fix

cephalin committed:
acrolinx

PRMerger12 committed:
Merge pull request #30019 from cephalin/links

cli link fix

cephalin committed:
cli link fix

Commit:
Merge pull request #30020 from cephalin/links

cli link fix

cephalin committed:
cli link fix

PRMerger18 committed:
Merge pull request #30021 from daveirwin1/master

Updated for 9.3.0

Commit:
Merge pull request #30022 from cephalin/links

cli link fix

cephalin committed:
cli link fix

PRMerger12 committed:
Merge pull request #30023 from cephalin/links

cli link fix

PRMerger17 committed:
Merge pull request #30017 from billmath/portal

updating

Commit:
Merge pull request #29977 from billmath/securityupdate

updating note

Commit:
updates

mayanknayar committed:
MVC changes for ASR multi-tenancy guidance

Splitting the guidance for multi-tenancy under relevant heads

mayanknayar committed:
Imporvements and link fixes

Acrolinx suggestions and link fixes

itechedit committed:
Merge pull request #28236 from SychevIgor/patch-1

Classic portal no longer supported by ML Workspace

GitHubber17 committed:
Merge pull request #28920 from MarkMcGeeAtAquent/table-api-quickstarts

Cosmos DB: Table API quickstart updates

itechedit committed:
Merge pull request #29909 from dominicbetts/cf-topology

CF configure article

rjagiewich committed:
Merge pull request #30011 from SychevIgor/apimanagedfix

images not used in article

rjagiewich committed:
Merge pull request #30012 from SychevIgor/scalingImagesNotUsing

images not used in article

GitHubber17 committed:
Merge pull request #29910 from dominicbetts/cf-faq-update

Update CF FAQ with map info

Commit:
Added required action.

Commit:
Added required action.

rjagiewich committed:
Merge pull request #30014 from SychevIgor/troubleshooting-creating-ml-workspace

images not used in article

Commit:
Added required action.

Commit:
Added required operation.

Commit:
Fix.

Commit:
corrected for feedback:

Commit:
Fix.

Commit:
Fix.

Commit:
Fix

PRMerger20 committed:
Merge pull request #29990 from cynthn/images-tut

Linux custom image tutorial freshness

Commit:
Fix

tynevi committed:
Merge pull request #29830 from mimig1/12122017redirect

Cosmos DB: Renaming documentdb files to sql-api.

GitHubber17 committed:
Merge pull request #29972 from Minewiskan/aasalias

Aasalias

Ja-Dunn committed:
Merge pull request #29304 from ShawnJackson/stream-analytics-out-of-order-and-late-events

edit pass: stream-analytics-out-of-order-and-late-events

billmath committed:
fixing link

mayanknayar committed:
Title metadata change

Metadata changes required

GitHubber17 committed:
Merge pull request #30016 from dominicbetts/cf-customize-updates

Updates to CF customize article

Commit:
quixkfix02

Commit:
Update analysis-services-data-providers.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Fix

GitHubber17 committed:
Merge pull request #30024 from georgewallace/autobug1144178

automation updates

Commit:
Fix

Commit:
Fix

v-alje committed:
Merge pull request #30037 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger13 committed:
Merge pull request #30031 from jimdial/patch-41

Added required action.

PRMerger14 committed:
Merge pull request #30036 from MarkusVi/quixkfix02

quickfix02

PRMerger20 committed:
Merge pull request #30035 from Minewiskan/patch-3

Update analysis-services-data-providers.md

Commit:
Fix

PRMerger14 committed:
Merge pull request #30033 from jimdial/patch-43

Added required action.

Commit:
Merge pull request #30032 from jimdial/patch-42

Added required action.

GitHubber17 committed:
Merge pull request #29850 from billmath/emergency

Emergency

Commit:
Merge pull request #30034 from jimdial/patch-44

Added required operation.

v-shils committed:
Merge pull request #30025 from mayanknayar/master

MVC changes for ASR multi-tenancy guidance
